src/Analysis/Engine/Impl/PythonAnalyzer.cs

```diff
@@ -216,6 +217,34 @@ public static PythonAnalyzer Create(IPythonInterpreterFactory factory, IPythonIn
             return entry;
         }
 
+        /// <summary>
+        /// Adds a new annotations file to the list of available annotation files and returns a ProjectEntry object.
+        /// 
+        /// This method is thread safe.
+        /// </summary>
+        /// <param name="moduleName">The name of the module; used to associate with imports</param>
+        /// <param name="filePath">The path to the file on disk</param>
+        /// <param name="cookie">An application-specific identifier for the module</param>
+        /// <returns>The project entry for the new module.</returns>
+        public IPythonProjectEntry AddModuleAnnotations(string moduleName, string filePath, Uri documentUri = null, IAnalysisCookie cookie = null) {
+            if (moduleName == null) {
+                throw new ArgumentNullException(nameof(moduleName));
+            }
+
+            moduleName += AnnotationsModuleSuffix;
+            var entry = new ProjectEntry(this, moduleName, filePath, documentUri, cookie);
```

Comment:
It looks like the body of the method is similar to `AddModule`:
```cs
public IPythonProjectEntry AddModuleAnnotations(string moduleName, string filePath, Uri documentUri = null, IAnalysisCookie cookie = null) {
    Check.ArgumentNotNull(nameof(moduleName), moduleName);
    return AddModule(moduleName + AnnotationsModuleSuffix, filePath, documentUri, cookie);
}
```

Reply:
Makes sense